op-signer/service/provider/utils.go

```diff
@@ -0,0 +1,49 @@
+package provider
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	kms "cloud.google.com/go/kms/apiv1"
+	"cloud.google.com/go/kms/apiv1/kmspb"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+)
+
+func PEMPublicKeyToAddress(keyName string) (*common.Address, error) {
+	ctx := context.Background()
+	client, err := kms.NewKeyManagementClient(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	request := kmspb.GetPublicKeyRequest{
+		Name: keyName,
+	}
+
+	result, err := client.GetPublicKey(ctx, &request)
+	if err != nil {
+		return nil, fmt.Errorf("kms get public key request failed: %w", err)
+	}
+	fmt.Printf("PEM:\n%s", result.Pem)
```

Comment:
Are these prints intended to be in the final output or just for debug? Similar for L40.

Reply:
debug

Reply:
If it's only for debug, then it should be commented out, right?

Reply:
As an internal tool, it is convenient and informative for the user to check whether each step is correct or not.